packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -619,7 +619,7 @@ contract VerifyOPCM is Script {
             string.concat(
                 "forge inspect ",
                 _contractName,
-                " abi --json | jq -r '.[] | select(.type == \"constructor\") | .inputs | map(.type) | join(\",\")'"
+                " abi --json | jq -r '.[] | select(.type == \"constructor\") | .inputs | map(if .type == \"tuple\" then \"(\" + (.components | map(.type) | join(\",\")) + \")\" else .type end) | join(\",\")'"
```

Comment:
Do we run VerifyOPCM in CI? Would be a good easy way to verify these changes work

Also, will we need to cherry pick this into an .rc-8 release?

Reply:
Sort of - we run it as part of the basic test suite but it doesn't hit these lines of code because we don't verify any contracts on Etherscan. I think we could probably do a fork test or something similar that actually ends up running this in CI but will require hard-coding an OPCM address and will cause issues if we change the OPCM in a way that isn't backwards compatible.